sass-graph.js

```diff
@@ -23,8 +23,11 @@ function resolveSassPath(sassPath, loadPaths) {
       return partialPath
     }
   }
-  var errMsg = "File to import not found or unreadable: " + sassPath;
-  throw errMsg;
+
+  var warnMsg = 'File to import not found or unreadable: assuming "' + scssPath +
```

Comment:
For the moment just make this a comment and return `false`. 
I'll add a proper logging interface later.